client-side/src/stories/DateTime/DateInputs.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import PropTypes from 'prop-types';
```

Comment:
the datetime isn't part of your task, please take it from existing files in main branch